lib/sqlalchemy/testing/suite/test_insert.py

```diff
@@ -533,27 +538,38 @@ def test_insert_w_floats(
         (
             "non_native_uuid",
             Uuid(native_uuid=False),
-            uuid.uuid4(),
+            uuid.uuid4,
         ),
         (
             "non_native_uuid_str",
             Uuid(as_uuid=False, native_uuid=False),
-            str(uuid.uuid4()),
+            lambda: str(uuid.uuid4()),
         ),
         (
             "generic_native_uuid",
             Uuid(native_uuid=True),
-            uuid.uuid4(),
+            uuid.uuid4,
             testing.requires.uuid_data_type,
         ),
         ("UUID", UUID(), uuid.uuid4(), testing.requires.uuid_data_type),
         (
             "LargeBinary1",
-            LargeBinary(),
+            LargeBinary,
             b"this is binary",
         ),
         ("LargeBinary2", LargeBinary(), b"7\xe7\x9f"),
-        ("PG BYTEA", BYTEA(), b"7\xe7\x9f", testing.only_on("postgresql")),
+        (
+            "BIT",
+            BIT(2),
+            lambda: BitString.from_int(1, length=2),
+            testing.only_on("postgresql+asyncpg"),
```

Comment:
OK I apologize that there's a "postgresql" test for BYTEA in this suite, and that really should not be there, however this led you to add more postgresql-specific tests here.   

in the case of BIT, it seems that even custom-asyncpg constructs right now are required, that would be a SQLAlchemy bug we would want to fix in 2.1.

So these new tests should be in tests/dialect/postgresql/test_types.py and I would propose a new suite called BITTest.  can we move the new tests there?  thanks.

Reply:
Yeah it didn't felt right tbh, but this is my first contribution so I wasn't sure where to put it. Just let me know once you ported BYTEA and I will change it accordingly.

Reply:
I think that's merged now.  you can rebase and just move what you have here right over there, it's the same format.  thanks!

Reply:
Is the conditional import of asyncpg still needed?

Reply:
yes, unless you are into a function that's enabled only for `+asyncpg`